articles/113_ros_time.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Clock and Time
+permalink: articles/clock_and_time.html
+abstract:
+  This article describes the ROS primatives to support programming which can run both in realtime as well as simulated time.
```

Comment:
I changed all usages to be 'real time' and added a note to differentiate 'real time' from requirements for deterministic computing requirements.